tpp-app/src/home/pages/CalendarScreen.js

```diff
@@ -32,6 +35,24 @@ const DayComponent = ({ date, state, marking, navigation }) => {
 }
 
 export const Calendar = ({navigation}) => {
+    const [calendarData, setCalendarData] = useState({});
+
+    //updates the data when you come back to the calendar screen
+    useEffect(() => {
+        const unsubscribe = navigation.addListener('focus', () => {
+            getCalendarData();
+        });
+        return unsubscribe;
+    }, [navigation]);
+
+    const getCalendarData = async () => {
+        // will need to choose the correct year depending on which year the user is looking at
+        const data = await getCalendarByYear(2022);
+        if(data != calendarData){
```

Comment:
They're both objects so JS will check if they are the same object in memory, so I think `data != calendarData` is just always true

Reply:
Oh right, I'll just remove it.